http-server/src/lib.rs

```diff
@@ -40,7 +40,7 @@ pub use access_control::{
 	hosts::{AllowHosts, Host},
 	AccessControl, AccessControlBuilder,
 };
-pub use jsonrpsee_types::{Error, TEN_MB_SIZE_BYTES};
+pub use jsonrpsee_types::*;
```

Comment:
Wouldn't it make more sense to re-export types in namespace like: `pub use jsonrpsee_types as types`?

Reply:
Yes I agree. I went ahead and did that so it's a lot more noise here now.